core/lib/generators/solidus/install/app_templates/authentication/solidus_auth_devise.rb

```diff
@@ -0,0 +1,12 @@
+# Skip if already in the gemfile
+if Bundler.locked_gems.dependencies['solidus_auth_devise']
+  say_status :skipping, 'solidus_auth_devise is already in the gemfile'
+else
+  bundle_command("add solidus_auth_devise")
+end
+
+if options[:auto_accept]
+  migrations_flag = options[:migrate] ? '--auto-run-migrations' : '--skip-migrations'
+end
+
+generate "solidus:auth:install #{migrations_flag}"
```

Comment:
I haven't tried, but I think it'll error unless `options[:auto_accept]` is given.

Reply:
TIL

```ruby
if false
  a = 1
end
a # => nil 🤯 
```